wagtail/wagtailimages/tests/tests.py

```diff
@@ -81,6 +81,11 @@ def test_image_tag_with_filters(self):
         self.assertTrue('width="400"' in result)
         self.assertTrue('height="300"' in result)
 
+    def test_image_tag_with_chained_filters(self):
+        result = self.render_image_tag(self.image, 'fill-200x200|height-150')
```

Comment:
@kaedroho Agreed. We probably want to do this either way, as `|` characters are probably just as bad as spaces in the "likelihood of trashing your filesystem if you don't escape them properly" stakes :-) Converting to dots probably makes sense, since that's basically what we're doing with the filenames already (`badger.max-100x100.greyscale.jpg`).